source/applications/advanced/color_balance.py

```diff
@@ -154,7 +154,6 @@ def _color_balance_calibration(camera, settings_2d):
     print("Starting color balance calibration")
     corrected_red_balance = 1.0
     corrected_blue_balance = 1.0
-    # TODO: Update algorithm to account for green as well.
```

Comment:
this should be in the other PR

Reply:
it is, note that all these PRs are following each other